### ISSUE TITLE
add doxygen dependency into README

### ISSUE THREAD

**Issue by yslan** [aprilnovak/nekRSDoc#2]
This is me being dumb. 
I forget to install doxygen, but the README should tell user do so.

**Comment by aprilnovak** [comment]
Yes, good point! Unless I made a mistake, the PR into nekRS doesn't have any doxygen stuff - when I said how to build these docs, I meant build them in the nekRS repository itself, not in this repo. Where are you building the documentation exactly?

**Comment by yslan** [comment]
I clone this repo and install it with make html.
Will try the NekRS/doc one.

**Comment by yslan** [comment]
NekRS/doc does work without doxygen. Will start from there.
Thanks

**Comment by aprilnovak** [comment]
For the sake of reviewing the nekRS PR, please build the documentation over on the nekRS repo - it's a little different than what's shown here and actually represents what will be merged.

Thanks!
